LuaMenu/widgets/api_user_handler.lua

```diff
@@ -174,7 +177,7 @@ local function GetUserComboBoxOptions(userName, isInBattle, userControl, showTea
 			end
 		end
 
-		if (not myPartyID) or myPartyID ~= userPartyID then
+		if (Configuration.hidePartySystem == false) and ((not myPartyID) or myPartyID ~= userPartyID) then
```

Comment:
`Configuration.hidePartySystem == false`
->
`not Configuration.hidePartySystem`

and avoid unnecessary parentheses:
`if not Configuration.hidePartySystem and (not myPartyID or myPartyID ~= userPartyID) then`

Reply:
Configuration.hidePartySystem == false is so that if they DONT define it explicitly as false, it still gets hidden. This was done so that ZK peeps dont suddenly wonder why their party system was hidden.

I might be wrong but isnt "not nil"  == true?

Reply:
I advise against logic where false and nil behave differently.

But what's confusing me.. I think you want hidePartySystem to behave the same way if it was not defined or explicitly defined as false. 
Wouldn't your original code hide it whenever it's not defined? So ZK peeps would be wondering the exact thing you were trying to avoid?

Reply:
Ah yes, you are correct, my naming was wrong